metakb/models/biomarkers.py

```diff
@@ -0,0 +1,29 @@
+"""Data models for Biomarkers and supporting concepts."""
+from dataclasses import dataclass
+from typing import List
+from enum import Enum
+
+
+class VariationType(Enum):
+    """A placeholder class that will be covered by the Variant Lexicon."""
+
+    PROTEIN_SUBSTITUTION = 1
```

Comment:
This is subject to change. I think we may end up with a multi-attribute model for variation type (e.g. protein sequence, substitution type in lieu of protein_substitution)